src/components/organisms/MessagesContainer.vue

```diff
@@ -1,9 +1,11 @@
 <template>
   <div class="flex">
-    <div class="w-full md:w-1/3 md:border md:border-r-0 md:border-gray">
-      <inbox />
+    <div class="w-full md:w-1/3 md:border md:border-r-0 md:border-gray md:h-96">
+      <div class="h-full md:overflow-auto">
+        <inbox />
+      </div>
     </div>
-    <div class="flex-auto hidden md:block md:border md:border-gray">
+    <div :class='[drawerOpen? "": "hidden", "bg-white w-screen h-screen absolute md:relative top-0 left-0 md:flex-auto md:block md:border md:border-gray md:h-96"]'>
```

Comment:
Use the v-show instead of the ternary operator.  :)

Reply:
I did the thing, great suggestion

Reply:
I had to revert this change because v-show can't be controlled with screen size